src/components/__tests__/slide_out_container.test.tsx

```diff
@@ -1,27 +1,51 @@
 // Copyright (c) 2018 Ultimaker B.V.
-import 'jest';
-import 'jsdom-global/register';
 import * as React from 'react';
 import { shallow } from 'enzyme';
+import { UnmountClosed } from 'react-collapse';
```

Comment:
This import should be a few lines below in the component list.

Reply:
I normally put external imports at the top, then our components under `// components`. react-collapse is an external import.